<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cronómetro</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_fondo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>480</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>700</width>
      <height>480</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>480</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>background.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_detectar">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>390</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>160</width>
      <height>160</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>AcadEref</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(128, 150, 56);
	color: rgb(255,255,255);  
	border-radius: 30px; 
	border: 2px solid #000;
	font: bold 9pt &quot;AcadEref&quot;;	
}

QPushButton:hover{
	background-color: rgb(73, 166, 39);
}</string>
    </property>
    <property name="text">
     <string>Iniciar detección</string>
    </property>
    <property name="shortcut">
     <string>Ctrl+D</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_terminar">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>390</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>160</width>
      <height>160</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>AcadEref</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(203, 50, 52);
	color: rgb(255,255,255);  
	border-radius: 30px; 
	border: 2px solid #000;
	font: bold 9pt &quot;AcadEref&quot;;	
}

QPushButton:hover{
	background-color: rgb(232, 3, 27);
}</string>
    </property>
    <property name="text">
     <string>Terminar</string>
    </property>
    <property name="shortcut">
     <string>Ctrl+D</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_estadisticas">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(242, 234, 220);
	color: rgb(0,0,0);  
	border-radius: 30px; 
	border: 2px solid #000;
	font: bold 9pt &quot;AcadEref&quot;;	
}

QPushButton:hover{
	background-color: rgb(172, 172, 172);
}</string>
    </property>
    <property name="text">
     <string>Ver estadísticas</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QFrame" name="frame_1">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(238, 193, 112);
border-radius: 10px; 
border: 2px solid #000;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label_frame_1">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>111</width>
          <height>81</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="titulo_numero_vuelta_1">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>6</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Tw Cen MT Condensed Extra Bold</family>
          <pointsize>9</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 9pt &quot;Tw Cen MT Condensed Extra Bold&quot;;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_2">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(238, 193, 112);
border-radius: 10px; 
border: 2px solid #000;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label_frame_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>111</width>
          <height>81</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="titulo_numero_vuelta_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>6</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Tw Cen MT Condensed Extra Bold</family>
          <pointsize>9</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 9pt &quot;Tw Cen MT Condensed Extra Bold&quot;;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_3">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(238, 193, 112);
border-radius: 10px; 
border: 2px solid #000;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label_frame_3">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>111</width>
          <height>81</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="titulo_numero_vuelta_3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>6</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Tw Cen MT Condensed Extra Bold</family>
          <pointsize>9</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 9pt &quot;Tw Cen MT Condensed Extra Bold&quot;;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_4">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(238, 193, 112);
border-radius: 10px; 
border: 2px solid #000;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label_frame_4">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>111</width>
          <height>81</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="titulo_numero_vuelta_4">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>6</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Tw Cen MT Condensed Extra Bold</family>
          <pointsize>9</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 9pt &quot;Tw Cen MT Condensed Extra Bold&quot;;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_5">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(238, 193, 112);
border-radius: 10px; 
border: 2px solid #000;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="label_frame_5">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>111</width>
          <height>81</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="titulo_numero_vuelta_5">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>5</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Tw Cen MT Condensed Extra Bold</family>
          <pointsize>9</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 9pt &quot;Tw Cen MT Condensed Extra Bold&quot;;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>741</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_titulo">
       <property name="font">
        <font>
         <family>Tw Cen MT Condensed Extra Bold</family>
         <pointsize>36</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
	font: 36pt &quot;Tw Cen MT Condensed Extra Bold&quot;;
	color: rgb(75, 54, 33);
	color: rgb(91, 57, 36);
}</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:55pt;&quot;&gt;CRONÓMETRO&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_cronometro_actual">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(75, 54, 33);
background-color: rgb(91, 57, 36);
font: 87 52pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);
border-radius: 30px; 
border: 2px solid #fff;</string>
       </property>
       <property name="text">
        <string>00:00.000</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>290</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 87 6pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>MINUTOS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>280</y>
      <width>91</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 87 6pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>Milésimas</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>290</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 87 6pt &quot;Arial Black&quot;;</string>
    </property>
    <property name="text">
     <string>SEGUNDOS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_linea_de_meta">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>390</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>160</width>
      <height>160</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>AcadEref</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(255, 249, 75);
	color: rgb(0,0,0);  
	border-radius: 30px; 
	border: 2px solid #000;
	font: bold 9pt &quot;AcadEref&quot;;	
}

QPushButton:hover{
	background-color: rgb(154, 154, 40);
}</string>
    </property>
    <property name="text">
     <string>Linea de meta</string>
    </property>
    <property name="shortcut">
     <string>Ctrl+D</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_abrir_calibrar">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>390</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>OpenHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	
	background-color: rgb(61, 55, 255);
	color: #fff;
	border-radius: 30px; 
	border: 2px solid #000;
	font: bold 9pt &quot;AcadEref&quot;;
}

QPushButton:hover{
	
	background-color: rgb(126, 150, 255);
}</string>
    </property>
    <property name="text">
     <string>Calibrar area</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
